build/helper/codegen_helper.py

```diff
@@ -205,17 +205,17 @@ def get_ctype_variable_declaration_snippet(parameter, parameters):
     snippet = parameter['ctypes_variable_name'] + ' = '
     if parameter['is_buffer']:
         if parameter['size']['mechanism'] == 'fixed':
-            snippet += '(' + 'ctypes_types.' + parameter['ctypes_type'] + ' * ' + str(parameter['size']['value']) + ')()'
+            snippet += '(' + 'visatype.' + parameter['ctypes_type'] + ' * ' + str(parameter['size']['value']) + ')()'
```

Comment:
I think there is still room for improvement in the metadata key names. But I can tackle that in a separate PR.